runtime/doc/pi_netrw.txt

```diff
@@ -1,29 +1,29 @@
 *pi_netrw.txt*  For Vim version 9.1.  Last change: 2024 Jul 13
 
-	    ------------------------------------------------
-	    NETRW REFERENCE MANUAL    by Charles E. Campbell
-	    ------------------------------------------------
+            ------------------------------------------------
+            NETRW REFERENCE MANUAL    by Charles E. Campbell
+            ------------------------------------------------
```

Comment:
Did you intentionally expand the tabs?

Reply:
No, I let git diff suppress them and sometimes stumble over it. Today is the day

Reply:
The tabs in all the other parts have also been expanded. So, this file has conflicts.